### PR TITLE
update users for internship

### PR DESCRIPTION
**Pull request by OlgaEvdokimova**

### CODE REVIEW

File: src/main/java/by/exadel/internship/controller/InternshipController.java [hunk 1]
```diff
@@ -94,4 +94,11 @@ public UserInternshipDTO updateInternship(@PathVariable(name = "internshipId") U
     public void assignUser(@RequestParam(name = "userIds") List<UUID> userIds, @PathVariable(name = "internshipId") UUID internshipId) {
         internshipService.assignUsers(userIds, internshipId);
     }
+
+    @SuperAdminAccessControl
+    @PutMapping("/{internshipId}/users/update")
```
Comment (SergeyKoval) [inline src/main/java/by/exadel/internship/controller/InternshipController.java]:
```suggestion
    @PutMapping("/{internshipId}/users/replace")
```

File: src/main/java/by/exadel/internship/controller/InternshipController.java [hunk 2]
```diff
@@ -94,4 +94,11 @@ public UserInternshipDTO updateInternship(@PathVariable(name = "internshipId") U
...
+    @ApiOperation("Update User to Internship")
+    public void updateUsersInInternship(@RequestParam(name = "userIds") List<UUID> userIds, @PathVariable(name = "internshipId") UUID internshipId) {
```
Comment (SergeyKoval) [inline src/main/java/by/exadel/internship/controller/InternshipController.java]:
```suggestion
    public void replaceUsersAssignedToInternship(@RequestParam(name = "userIds") List<UUID> userIds, @PathVariable(name = "internshipId") UUID internshipId) {
```

File: src/main/java/by/exadel/internship/controller/InternshipController.java [hunk 3]
```diff
@@ -94,4 +94,11 @@ public UserInternshipDTO updateInternship(@PathVariable(name = "internshipId") U
...
+        internshipService.updateUsers(userIds, internshipId);
```
Comment (SergeyKoval) [inline src/main/java/by/exadel/internship/controller/InternshipController.java]:
```suggestion
        internshipService.replaceUsersAssignedToInternship(userIds, internshipId);
```

Comment (OlgaEvdokimova) [reply src/main/java/by/exadel/internship/controller/InternshipController.java]: Done

File: src/main/java/by/exadel/internship/controller/InternshipController.java (hunk 1 above)
Comment (OlgaEvdokimova) [reply src/main/java/by/exadel/internship/controller/InternshipController.java]: Thanks for suggestions. Done

**Review by ShirshovAlex — DISMISSED** [review]

File: src/main/java/by/exadel/internship/controller/InternshipController.java (hunk 2 above)
Comment (OlgaEvdokimova) [reply src/main/java/by/exadel/internship/controller/InternshipController.java]: Done

**Review by SergeyKoval — DISMISSED** [review]

**Review by ShirshovAlex — APPROVED** [review]

**Review by Fenydude — APPROVED** [review]

**Pull request merged**